handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: drush clear cache
```

Comment:
@benjifisher I like my handlers lowercase, so how about `clear the drush cache` or simply `clear drush cache`?